src/main/java/creativei/service/impl/InquiryServiceImpl.java

```diff
@@ -35,16 +36,21 @@
     }
 
     @Override
-    public List<Inquiry> getByStatus(InquiryStatus status) {
-        return inquiryDao.findByInquiryStatus(status);
+    public List<Inquiry> getByStatus(InquiryStatus status) throws NoDataAvailable{
+        List<Inquiry> inquiries= inquiryDao.findByInquiryStatus(status);
+        if(inquiries.size()==0){
+            throw new NoDataAvailable(ExceptionType.DATA_NOT_AVAILABLE.getMessage());
```

Comment:
Send consistent error back for similar exceptions

Reply:
solved in commit d5326cd